acc_test/data/valid_with_resizable_opensearch_cluster.tf

```diff
@@ -20,7 +20,7 @@ resource "instaclustr_cluster" "resizable_cluster" {
   }
   bundle {
     bundle  = "OPENSEARCH"
-    version = "1.2.4"
+    version = "1.3.1"
```

Comment:
with this version change, can you please enable this 1.3.1 for [tf test account ](https://instaclustr.atlassian.net/wiki/spaces/IDS/pages/1623425094/Terraform+Provider+Test+Account)

this is preemptive action for subsequent opensearch version bump where 1.3.1 becomes disabled, which will fail this test

Reply:
That's a very good reminder. Thanks Long